pkg/eden/eden.go

```diff
@@ -339,6 +339,40 @@ func StatusEServer() (status string, err error) {
 	return state, nil
 }
 
+//StartEVEVBox function run EVE in VirtualBox
+func StartEVEVBox(vmName string, vBoxCpus int, vBoxMem int) (err error) {
+	commandArgsString := fmt.Sprintf("createvm --name %s --register ", vmName)
+	if err = utils.RunCommandWithLogAndWait("VBoxManage", defaults.DefaultLogLevelToPrint, strings.Fields(commandArgsString)...); err != nil {
+		log.Fatalf("VBoxManage error for command %s %s", commandArgsString, err)
+	}
+	commandArgsString = fmt.Sprintf("modifyvm %s --cpus %d --memory %d --vram 16 --nested-hw-virt on --ostype Ubuntu_64  --mouse usbtablet --graphicscontroller vmsvga --boot1 disk --boot2 net", vmName, vBoxCpus, vBoxMem)
+	if err = utils.RunCommandWithLogAndWait("VBoxManage", defaults.DefaultLogLevelToPrint, strings.Fields(commandArgsString)...); err != nil {
+		log.Fatalf("VBoxManage error for command %s %s", commandArgsString, err)
+	}
+
+	commandArgsString = fmt.Sprintf("storagectl %s --name \"SATA\" --add sata --bootable on --hostiocache on", vmName)
+	if err = utils.RunCommandWithLogAndWait("VBoxManage", defaults.DefaultLogLevelToPrint, strings.Fields(commandArgsString)...); err != nil {
+		log.Fatalf("VBoxManage error for command %s %s", commandArgsString, err)
+	}
+
+	commandArgsString = fmt.Sprintf("storageattach %s  --storagectl \"SATA\" --port 0 --device 0 --type hdd --medium dist/default-images/eve/live.raw.vdi", vmName)
+	if err = utils.RunCommandWithLogAndWait("VBoxManage", defaults.DefaultLogLevelToPrint, strings.Fields(commandArgsString)...); err != nil {
+		log.Fatalf("VBoxManage error for command %s %s", commandArgsString, err)
+	}
+
+	commandArgsString = fmt.Sprintf("modifyvm %s  --nic1 natnetwork --nat-network1 natnet1 --cableconnected1 on --natpf1 ssh,tcp,,2222,,22", vmName)
```

Comment:
Can you please use ports from hostfwd here? The same as [here](https://github.com/lf-edge/eden/blob/508d5df565d4b88249cdef0e9989451c72bf9f15/pkg/eden/eden.go#L356).